docs/ref/models/querysets.txt

```diff
@@ -1844,34 +1844,42 @@ they query the database each time they're called.
 .. method:: get(**kwargs)
 
 Returns the object matching the given lookup parameters, which should be in
-the format described in `Field lookups`_.
+the format described in `Field lookups`_. Normally you should use lookups
+that are guaranteed unique by the database, such as the primary key or fields
+in a unique constraint. For example::
 
-``get()`` raises :exc:`~django.core.exceptions.MultipleObjectsReturned` if more
-than one object was found. The
-:exc:`~django.core.exceptions.MultipleObjectsReturned` exception is an
-attribute of the model class.
+    Entry.objects.get(id=1)
 
-``get()`` raises a :exc:`~django.db.models.Model.DoesNotExist` exception if an
-object wasn't found for the given parameters. This exception is an attribute
-of the model class. Example::
+If ``get()`` finds more than one object, it raises a
+:exc:`Model.MultipleObjectsReturned <django.db.models.Model.MultipleObjectsReturned>`
+exception::
 
-    Entry.objects.get(id='foo') # raises Entry.DoesNotExist
+    Entry.objects.get(name='A Duplicated Name') # raises Entry.MultipleObjectsReturned
 
-The :exc:`~django.db.models.Model.DoesNotExist` exception inherits from
-:exc:`django.core.exceptions.ObjectDoesNotExist`, so you can target multiple
-:exc:`~django.db.models.Model.DoesNotExist` exceptions. Example::
+If ``get()`` doesn't find any object, it raises a
+:exc:`Model.DoesNotExist <django.db.models.Model.DoesNotExist>` exception::
+
+    Entry.objects.get(id='non-existent-id') # raises Entry.DoesNotExist
+
+Both these exception classes are attributes of the model class, and specific to
+that model. If you want to handle such exceptions from several ``get()`` calls,
+you can use their generic base classes. For example, you can use
+:exc:`django.core.exceptions.ObjectDoesNotExist`  to handle
+:exc:`~django.db.models.Model.DoesNotExist` exceptions from multiple models.
+Example::
```

Comment:
Drop `Example::` and move the `::` to the previous sentence which already starts with `For example…`.